reflex/testing.py

```diff
@@ -544,37 +548,43 @@ def frontend(
         if self.frontend_url is None:
             raise RuntimeError("Frontend is not running.")
         want_headless = False
-        options: ArgOptions | None = None
         if os.environ.get("APP_HARNESS_HEADLESS"):
             want_headless = True
         if driver_clz is None:
             requested_driver = os.environ.get("APP_HARNESS_DRIVER", "Chrome")
             driver_clz = getattr(webdriver, requested_driver)
-            options = getattr(webdriver, f"{requested_driver}Options")()
+            if driver_options is None:
+                driver_options = getattr(webdriver, f"{requested_driver}Options")()
```

Comment:
seems we can get rid of these now that each driver has a similar bit of code below.